console.error("HALP!")
console.log("PLUTO!")
console.warn("TRY AGAIN")
